triggers.md

```diff
@@ -96,6 +98,13 @@ The following example illustrates the basic idea:
 
 (The use of `<script>` is basically because only a few elements allow the HTML parser to enter RCDATA mode, and `<script>` has already been used in this way. This doesn't necessarily mean that these rules should respect the `script-src` CSP directive.)
 
+For use cases where it is preferable to provide speculation rule sets without modifying a document, we introduce the `Speculation-Rules` HTTP header. The header's values are URLs to external resources containing speculation rule sets. The rules are expressed in the same JSON object format as the inline case. The MIME type of the resource must be `application/speculationrules+json`.
```

Comment:
There is a sentence above this that probably should be modified:

> Currently, like import maps, script tags are only used for specifying speculation rules inline; future extensions may allow external ones.

Reply:
Done